dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -0,0 +1,67 @@
+[supervisord]
+logfile_maxbytes=1MB
+logfile_backups=2
+nodaemon=true
+
+[program:start.sh]
+command=/usr/bin/start.sh
+priority=1
+autostart=true
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n
+priority=2
+autostart=false
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+{# If our configuration has VLANs... #}
+{% if VLAN %}
+{# Count how many VLANs require a DHCP relay agent... #}
+{% set num_relays = { 'count': 0 } %}
+{% for vlan_name in VLAN %}
+{% if VLAN[vlan_name]['dhcp_servers'] %}
+{% set _dummy = num_relays.update({'count': num_relays.count + 1}) %}
+{% endif %}
+{% endfor %}
+{# If one or more of the VLANs require a DHCP relay agent... #}
+{% if num_relays.count > 0 %}
+[group:isc-dhcp-relay]
+programs=
+{%- set add_preceding_comma = { 'flag': False } -%}
+{%- for vlan_name in VLAN -%}
+{%- if VLAN[vlan_name]['dhcp_servers'] -%}
+{%- if add_preceding_comma.flag %},{% endif -%}
+{%- set _dummy = add_preceding_comma.update({'flag': True}) -%}
+isc-dhcp-relay-{{ vlan_name }}
+{%- endif %}
+{% endfor %}
+
+
+{# Create a program entry for each DHCP relay agent instance #}
+{% for vlan_name in VLAN -%}
+{%- if VLAN[vlan_name]['dhcp_servers'] -%}
+[program:isc-dhcp-relay-{{ vlan_name }}]
+command=/usr/sbin/dhcrelay -d -c 1 -a %%h:%%p %%P -i {{ vlan_name }}
```

Comment:
>/usr/sbin/dhcrelay -d -c 1 [](start = 8, length = 26)

we are going to forward the already relayed packet? 

-m append|replace|forward|discard ?

Reply:
how do you control the behavior for already relayed packet?

---
In reply to: [142012491](https://github.com/Azure/sonic-buildimage/pull/999#discussion_r142012491) [](ancestors = 142012491)

Reply:
I am preventing the relay of previously-relayed packets by limiting the max-hop-count to 1 (`-c 1`), under the assumption that we should only ever relay requests from devices immediately below the ToR (i.e., hop count of 0). If this assumption is incorrect, I will investigate the behavior of the `-m` flag.